Java/other/LongestSubArrayHavingSumk.java

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Add a beginner introduction of the program with the space and time complexity of your approach.